src/DetailsView/components/test-status-choice-group.scss

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../common/styles/colors.scss';
+
+.radio-button-group {
+    float: left;
+    line-height: 24px;
+
+    .radio-label {
+        font-size: 14px;
+        padding: 0 0 0 26px;
+        display: inline-block;
+    }
+
+    :global {
```

Comment:
out of curiosity: what `:global` stand for?

Reply:
This is to say that the following classes are not declared in this module. As a result, these are not modified by css-loader. This is beneficial in the scenarios we depend on certain class names being X or Y from components we don't control (Office Fabric).

Reply:
I see... so you can't import them on your components neither, right?

Reply:
Correct!